components/sidebar-icons/icons.tsx

```diff
@@ -0,0 +1,243 @@
+import React from 'react'
+
+export const All = () => {
```

Comment:
Either:

1. Name all of them like an icon, for example: `AllIcon` / `OpenTelemetryIcon` etc. 

OR

2. Export all of these in a single object like this:
```
export const SidebarIcons = {
  All,
  OpenTelemetry,
  ...
}
 ```

These Icons as standalone components without the "Icon" in their name are misleading nomenclature otherwise as components.